tests/frontend/tests/test_threebot_deployer.py

```diff
@@ -0,0 +1,130 @@
+import pytest
+import urllib.request
+import os
+from jumpscale.loader import j
+from random import randint
+from tests.frontend.tests.base_tests import BaseTest
+from tests.frontend.pages.wallets.wallets import Wallets
+from tests.frontend.pages.threebot_deployer.threebot_deployer import ThreebotDeployer
+
+
+@pytest.mark.integration
+class ThreebotDeployerTests(BaseTest):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.info("Import a wallet")
+        cls.wallet_name = "demos_wallet"
+        cls.wallet_secret = os.environ.get("WALLET_SECRET")
+        if not cls.wallet_secret:
+            raise Exception("Please add WALLET_SECRET as environment variables")
+
+        j.clients.stellar.get(cls.wallet_name, network="STD", secret=cls.wallet_secret)
```

Comment:
Please get the wallet then change the secret as we do in vdc tests https://github.com/threefoldtech/js-sdk/blob/development/tests/sals/vdc/vdc_base.py#L37